atomata/capstone_WebClient
# Login Page Tests

### Summary 
New tests for the Login page. Also fixed a bug with tests crashing because of a lack of router instances.
### Details
- Added 4 new tests for login
- Fixed issue with index and login tests crashing from having no router instance
- There are still 2 tests that fail (PreviewOverlay.test.tsx)
### How to test?
1. Open project
2. Run tests using "yarn test"
3. login.test.tsx should pass
4. index.test.tsx should also be passing now
